sys/net/gnrc/network_layer/ipv6/ext/frag/gnrc_ipv6_ext_frag.c

```diff
@@ -291,13 +300,13 @@ static void _snd_buf_del(gnrc_ipv6_ext_frag_send_t *snd_buf)
     snd_buf->pkt = NULL;
 }
 
-static void _snd_buf_free(gnrc_ipv6_ext_frag_send_t *snd_buf)
+static void _snd_buf_free(gnrc_ipv6_ext_frag_send_t *snd_buf, int errno)
 {
     if (snd_buf->per_frag) {
         gnrc_pktbuf_release(snd_buf->per_frag);
```

Comment:
Should we also use `gnrc_pktbuf_release_error()` here?

Reply:
`per_frag` is derived from `pkt`, so I don't think that is necessary. A subscriber would then only get duplicate messages:

https://github.com/RIOT-OS/RIOT/blob/3234b918e361d273b9cfef3c123653b174f0abc0/sys/net/gnrc/network_layer/ipv6/ext/frag/gnrc_ipv6_ext_frag.c#L112-L127